src/extractMeta.js

```diff
@@ -1,26 +1,9 @@
 /* eslint-disable security/detect-object-injection */
-import { fromCamelToSentence } from './utils';
+import extract from './extract';
+import * as ElementBlock from './ElementBlock';
 
 const extractMetadata = (props = {}) => {
-    const extraction = {};
-    Object.keys(props).forEach(key => {
-        if (!props[key]) {
-            return;
-        }
-        const propType = props[key].type ? props[key].type : props[key];
-        const label =
-            props[key].label === undefined
-                ? fromCamelToSentence(key)
-                : props[key].label;
-        extraction[key] = {
-            ...propType._meta,
-            label,
-            propName: key,
-            isPrivate: props[key].isPrivate,
-            tooltip: props[key].tooltip
-        };
-    });
-    return extraction;
+    return extract({ ...ElementBlock.configSchema, ...props })
```

Comment:
I wonder if there is a way to tag this config in the extracted meta so that we can show it differently in the SD. Like a special section on the edit panel containing these common props.

Reply:
@santios-volusion 

Yes, I think something like that would be good. I was thinking of wrapping it in a `shape` proptype, like this:

```
extract({ standardProps: {
  label: "Standard Block Props",
  type: ElementPropTypes.shape(ElementBlock.configSchema)
}, ...props })
```

But it might be nice to have something more explicit than just an arbitrarily named property.